Tests/ValidationTests.swift

```diff
@@ -176,10 +174,10 @@ class ContentTypeValidationTestCase: BaseTestCase {
         // Then
         XCTAssertNotNil(error)
 
-        if let error = error {
-            XCTAssertEqual(error.domain, ErrorDomain)
-            XCTAssertEqual(error.code, ErrorCode.contentTypeValidationFailed.rawValue)
-            XCTAssertEqual(error.userInfo[ErrorUserInfoKeys.ContentType] as? String, "application/xml")
+        if let error = error as? AFError, let contentType = error.responseContentType, let acceptableTypes = error.acceptableContentTypes {
+            XCTAssertTrue(error.isUnacceptableContentType)
+            XCTAssertEqual(contentType, "application/xml")
+            XCTAssertEqual(acceptableTypes[0], "application/octet-stream")
         } else {
             XCTFail("error should not be nil")
         }
```

Comment:
This will crash if `acceptableTypes` is empty. Could we get a check for the index as well as blow out the `XCTFail` explanation to explain the other cases where this could fail?